infrastructure/resource_group_level/modules/trade-finance-manager-api.bicep

```diff
@@ -0,0 +1,251 @@
+param location string
+param environment string
+param containerRegistryName string
+param appServicePlanEgressSubnetId string
+param appServicePlanId string
+param privateEndpointsSubnetId string
+param cosmosDbAccountName string
+param cosmosDbDatabaseName string
+param logAnalyticsWorkspaceId string
+param externalApiHostname string
+param portalApiHostname string
+param dtfsCentralApiHostname string
+param storageAccountName string
+
+var dockerImageName = '${containerRegistryName}.azurecr.io/trade-finance-manager-api:${environment}'
+var dockerRegistryServerUsername = 'tfs${environment}'
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureSettings object = {
+
+}
+
+// These values are taken from an export of Configuration on Dev (& validating with staging).
+@secure()
+param additionalSecureSettings object = {
+  DOCKER_REGISTRY_SERVER_PASSWORD: 'test-value'
+  // TODO:FN-737 UKEF_INTERNAL_NOTIFICATION is new and necessary
+  UKEF_INTERNAL_NOTIFICATION: 'test-value'
+  // TODO:FN-737 The following 5 secrets are from the APIM changes
+  DTFS_CENTRAL_API_KEY: 'test-value'
+  EXTERNAL_API_KEY: 'test-value'
+  JWT_VALIDATING_KEY: 'test-value'
+  PORTAL_API_KEY: 'test-value'
+  TFM_API_KEY: 'test-value'
+}
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureConnectionStrings object = {
+  // NOTE that CORS_ORIGIN is not present in the variables exported from dev or staging
+  CORS_ORIGIN: 'test-value'
+  UKEF_TFM_API_SYSTEM_KEY: 'test-value'
+  UKEF_TFM_API_REPORTS_KEY: 'test-value'
+  // TODO:FN-429 Note that this has a value like https://tfs-dev-tfm-fd.azurefd.net 
+  // while in the CLI it is injected as a secret, we can probably calculate it from the Front Door component.
+  TFM_URI: 'test-value'
+  AZURE_NUMBER_GENERATOR_FUNCTION_SCHEDULE: 'test-value'
+  JWT_SIGNING_KEY: 'test-value' // NOTE - in the export this appears to be a slot setting.
+}
+
+// These values are taken from an export of Connection strings on Dev (& validating with staging).
+@secure()
+param additionalSecureConnectionStrings object = {
+  GOV_NOTIFY_EMAIL_RECIPIENT: 'test-value'
+}
+
+
+// https://learn.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16
+var azureDnsServerIp = '168.63.129.16'
+
+// These values are hardcoded in the CLI scripts
+var settings = {
+  WEBSITE_DNS_SERVER: azureDnsServerIp
+  WEBSITE_VNET_ROUTE_ALL: '1'
+  PORT: '5000'
+  WEBSITES_PORT: '5000'
+}
+
+// These values are taken from an export of Configuration on Dev (& validating with staging).
+var dtfsCentralApiUrl = 'http://${dtfsCentralApiHostname}'
+// Note that in the CLI script, http was used, but the value in the exported config was https.
```

Comment:
in reality, all URLs should be HTTPs. can we change Central-API to follow the same approach?

Reply:
I assume so! @abhi-markan is looking into this. I've created FN-741 to track it.